cmp/tab/TabContainerModel.ts

```diff
@@ -20,85 +87,50 @@ import {TabModel} from './TabModel';
  */
 export class TabContainerModel extends HoistModel {
 
-    /** @member {TabModel[]} */
     @managed
     @observable.ref
-    tabs = null;
-
-    /** @member {?string} */
-    route;
-
-    /** @member {string} */
-    @observable activeTabId;
-
-    /** @member {boolean|Object} */
-    switcher;
-
-    /** @member {boolean} */
-    track;
+    tabs: TabModel[] = null;
 
-    /** @member {RenderMode} */
-    renderMode;
+    @observable
+    activeTabId: string;
 
-    /** @member {RefreshMode} */
-    refreshMode;
+    route: string;
+    defaultTabId: string;
+    switcher: TabSwitcherProps;
+    track: boolean;
+    renderMode: RenderMode;
+    refreshMode: RefreshMode;
+    emptyText: ReactNode;
+    provider: PersistenceProvider;
 
-    /** @member {(string|ReactNode)} */
-    emptyText;
-
-    /** @member {RefreshContextModel} */
     @managed
-    refreshContextModel;
+    refreshContextModel: RefreshContextModel;
 
-    _lastActiveTabId;
+    private _lastActiveTabId;
```

Comment:
Should we lose the undescore here?

Reply:
done -- will be more aggressive with those